### PR TITLE
A bit of restructure

### PR DESCRIPTION
**Pull request by pltanton**
Here some restructure of repository, for my taste is better when you've got files not in one dirrectory. Also  I replaced `satrtup.sh` script with systemd service, it makes startup of vm a little bit faster.

### CODE REVIEW

**Comment by steveswinsburg** [comment]
Thanks for this. I can probably take a bit of this but not all of it. Some of the stuff like the port number change and the create-schema would be specific to your case. Plus the reformatting of the readme.

But I love the systemd service. I just pushed up a commit that has made this out of sync, could you rebase and remove some of those extra changes and we can work on getting this in?

**Comment by pltanton** [comment]
Here it is.

File: scripts/install.sh [hunk 1]
```diff
@@ -53,15 +54,15 @@ chown oracle:oinstall -R /opt
...
+echo "export ORACLE_BASE=/opt/oracle" >> /home/oracle/.bashrc \
+ && echo "export ORACLE_HOME=/opt/oracle/product/12.1.0.2/dbhome_1" >> /home/oracle/.bashrc \
+ && echo "export ORACLE_SID=orcl" >> /home/oracle/.bashrc \
+ && echo "export PATH=\$PATH:\$ORACLE_HOME/bin" >> /home/oracle/.bashrc
 
 echo 'INSTALLER: Environment variables set'
 
 # install Oracle
-su -l oracle -c "yes | /vagrant/database/runInstaller -silent -showProgress -ignorePrereq -waitforcompletion -responseFile /vagrant/db_install.rsp"
+su -l oracle -c "yes | /vagrant/database/runInstaller -silent -showProgress -ignorePrereq -waitforcompletion -responseFile /vagrant/ora-responce/db_install.rsp"
```
Comment (steveswinsburg) [inline scripts/install.sh:65]: Looks great. The only thing is the spelling of response, with an s. Then I'll merge this.

Cheers!

Comment (pltanton) [reply scripts/install.sh:65]: Lol, feeling dumb.

**Review by steveswinsburg — APPROVED** [review]
Thanks!

**Pull request merged**